hack/run-test-e2e-serial.sh

```diff
@@ -2,6 +2,20 @@
 
 source hack/common.sh
 
+# we expect the lane to run against a already configured cluster
+SETUP="${SETUP:-false}"
+TEARDOWN="${TEARDOWN:-false}"
```

Comment:
I think the default should be keeping the cluster intact, so ```true```

Reply:
you mean they should go in pairs? if we don't do setup (and we don't want to do that _by default_), we should not do teardown either

Reply:
My bad, I Missed the ```SETUP``` variable above